site/app/libraries/database/DatabaseQueries.php

```diff
@@ -818,6 +824,17 @@ public function isStaffPost($author_id) {
         return intval($this->course_db->rows()[0]['user_group']) <= 3;
     }
 
+    public function isInstructorOrFullAccess($author_id) {
+        $this->course_db->query("SELECT user_group FROM users WHERE user_id=?", [$author_id]);
+        return intval($this->course_db->rows()[0]['user_group']) <= 2;
```

Comment:
```suggestion
        return intval($this->course_db->row()['user_group']) <= 2;
```

Reply:
It would also be good to use a named constant here instead of an arbitrary integer constant which is not as readable.